extensions/amp-video/0.1/amp-video.md

```diff
@@ -156,6 +156,13 @@ Indicates the album/collection the video was taken from, specified as a string.
 
 Indicates the name/title of the video, specified as a string. If not provided, the Media Session API helper uses either the `aria-label` attribute or falls back to the page's title.
 
+[filter formats="stories"]
+
+#### cache
+
+Indicates the Google video cache should store and serve the video by adding `cache="google"`. The video cache will fetch and store the video contents periodically, reducing serving costs for videos, and generating transcodes with different quality settings that adapt the bitrate to the network conditions.
```

Comment:
Should we use this snippet even within `amp-story.md`? I got a bit lost with the origin/cache/video cache wording, this one looks better

Reply:
We need to keep the first sentence on `amp-story.md` so we exaplain it's on `amp-video` but I can make the rest the same as here